features/networking/multus.feature

```diff
@@ -417,3 +417,35 @@ Feature: Multus-CNI related scenarios
     And the output should contain "net2"
     And the output should contain 2 times:
       | macvlan mode bridge |
+
+
+  # @author weliang@redhat.com
+  # @case_id OCP-25676
+  @admin
+  Scenario: Supported runtimeConfig/capability for MAC/IP
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    And evaluation of `project.name` is stored in the :project_name clipboard
```

Comment:
@weliang1 if your test does not change project context during the test, then you don't need to save the project name to the clipbord.

Reply:
@pruan-rht Thanks, delete the line of save the project name to the clipbord

Reply:
@pruan-rht I put that line back, otherwise the net-attach-def configuration was defined in default project not the new project name

Reply:
@pruan-rht Got advise from @anuragthehatter, I remove that line again, and it works when use <%= project.name %>, thanks @anuragthehatter !

Reply:
@weliang1 the original PR has been merged...pls submit another PR.